### DISCUSSION TITLE
Renovate to fail running a project without renovate.json

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [renovatebot/renovate#15552] [Request Help]
Hello, I have the following configuration and renovate setup:

* GitLab - Renovate self-hosted
* Projects run the Renovate on their own pipeline
* We have custom configurations that are extended by each project depending on the package manager as `npm` or `docker`

When a new project is onboarding and forgets to add the `renovate.json` to the default branch and renovate runs it will create a first PR adding a `renovate.json` instead of failing. 

Is it possible to make the Renovate execution fail if there is not a `renovate.json` file? Otherwise, I can add a check to the pipeline to make it fail to check if the file exists.

**Comment by rarkins** [comment]
If you configure onboarding=false then it will simply skip over repos without config

**Reply by viceice** [reply]
but don't fail, which is by design and won't changed in future.

**Reply by juancarlosjr97** [reply]
Great, thank you, guys. I knew I saw it but I was not sure which configuration was it! Awesome!
